Docs/Entities.md

```diff
@@ -0,0 +1,31 @@
+# Entities
+
+### User
+- ID ```number```
+- First Name ```string(25)```
+- Last Name ```string(25)```
+- Email ```string(100)```
+- Password ```string(100)```
+- Country ```string```
+
+### Skill
+- ID ```number```
+- Title ```string(50)```
```

Comment:
What do you mean by title?

Reply:
It's basically the name, such as JavaScript, Angular and so on...